routes/api/contacts.js

```diff
@@ -1,25 +1,141 @@
-const express = require('express')
+const express = require("express");
+const { Contacts } = require("../schema");
+const router = express.Router();
+const { validateJoi } = require("./validation");
 
-const router = express.Router()
+router.get("/", async (__, res) => {
+  const contacts = await Contacts.find();
+  try {
+    res.json({
+      status: "success",
+      code: 200,
+      data: contacts,
+    });
+  } catch (error) {
+    res.status(400).json({
+      status: "error",
+      code: 400,
+      message: "Bad Request",
+    });
+  }
+});
 
-router.get('/', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.get("/:contactId", async (req, res) => {
+  const { contactId } = req.params;
+  try {
+    const contact = await Contacts.findById(contactId);
+    res.json({
+      status: "success",
+      code: 200,
+      data: contact,
+      message: "Contact has been found",
+    });
+  } catch (error) {
+    res.status(400).json({
+      status: "error",
+      code: 400,
+      message: "Bad Request",
+    });
+  }
+});
 
-router.get('/:contactId', async (req, res, next) => {
-  res.json({ message: 'template message' })
-})
+router.post("/", async (req, res) => {
+  const body = req.body;
+  const contact = new Contacts(body);
+  const validate = validateJoi(contact);
+  try {
+    await validate.value.save();
```

Comment:
```suggestion
    await contact.save();
```
Jeśli użyjesz mojej sugestii o której pisałem wcześniej odnośnie walidacji wówczas tu się tak zmieni